invenio_files_rest/serializer.py

```diff
@@ -224,14 +224,7 @@ def schema_from_context(context):
 
 
 def _format_args():
-    try:
-        pretty_format = \
-            current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and \
-            not request.is_xhr
-    except RuntimeError:
-        pretty_format = False
-
-    if pretty_format:
+    if request and request.args.get('prettyprint'):
```

Comment:
is this the same used in records-rest? i.e. ``prettyprint`` and not ``pretty``

Reply:
yes, in records-rest there's [`prettyprint`](https://github.com/inveniosoftware/invenio-records-rest/blob/master/invenio_records_rest/serializers/json.py#L25)